zcash_client_backend/src/data_api/testing/transparent.rs

```diff
@@ -175,8 +190,10 @@ where
     // and total balance, we should perform additional checks against available balance;
     // we use minconf 0 here because all transparent funds are considered shieldable,
     // irrespective of confirmation depth.
```

Comment:
Please update this comment. The tests now do distinguish between available and total (or pending) balances. They still don't distinguish between pending transparent change and pending transparent non-change.